xarray/computation/apply_ufunc.py

```diff
@@ -141,8 +141,13 @@ def __repr__(self):
         return f"{type(self).__name__}({list(self.input_core_dims)!r}, {list(self.output_core_dims)!r})"
 
     def __str__(self):
-        lhs = ",".join("({})".format(",".join(dims)) for dims in self.input_core_dims)
-        rhs = ",".join("({})".format(",".join(dims)) for dims in self.output_core_dims)
+        comma_separated = ",".join
+        lhs = comma_separated(
```

Comment:
not sure this got _simpler_, but ok!

Reply:
Not sure either.

The only benefit might be that the intent shows clearly: separate with commas.